vhdbuilder/prefetch/pkg/cni/cni.go

```diff
@@ -0,0 +1,72 @@
+package cni
+
+import (
+	"bytes"
+	_ "embed"
+	"fmt"
+	"os"
+	"strings"
+	"text/template"
+
+	"github.com/Azure/agentbaker/vhdbuilder/prefetch/pkg/component"
+)
+
+var (
+	//go:embed cni-prefetch.sh.gtpl
+	prefetchScript         string
+	prefetchScriptTemplate = template.Must(template.New("cniprefetch").Parse(prefetchScript))
+
+	// this represents the CNI-related container images and their respective binaries
+	// which are in-scope for prefetch optimization
+	prefetchList = CNIPrefetchList{
```

Comment:
any chance we could put this in components.json? Worried people won't know about this and will forget to update it as things change.

Reply:
fair point, will also try to address "prefetchable" binaries per container image version as well with this shift